ag_data/formulas/ingestion_engine.py

```diff
@@ -0,0 +1,33 @@
+from ag_data import models
+from ag_data.formulas.library.system import mercury_formulas as hgFormulas
+
+
+class MeasurementIngestionEngine:
+
+    processingFormulas = {
+        0: hgFormulas.fPass,
+        2: hgFormulas.fMercurySimpleTemperatureSensor,
+        4: hgFormulas.fMercuryDualTemperatureSensor,
+        6: hgFormulas.fMercuryFlowSensor,
+    }
```

Comment:
If these `0,2,4,6` values are predefined numbers mapped to each function, then it feels more like data than code for me. And I also expect end-users will touch this lines to add sensors on their own.
Considering that, I guess it would be better that we have this file as a separate python file instead of the middle of an implementation. It's more accessible and more manageable.

Reply:
Yes, I would love to make this change and create a dedicated file to handle the mapping.

Reply:
We can continue with this in #378, as this change will be bigger than originally anticipated.